src/main/java/com/redfin/sitemapgenerator/SitemapGenerator.java

```diff
@@ -163,6 +165,38 @@ THIS getThis() {
 		return outFiles;
 	}
 	
+	/**
+	 * Writes out the sitemaps as strings
+	 * @return a list of XML-formatted strings
+	 */
+	public List<String> writeAsString() {

```

Comment:
Please add a unit test for this. Especially add a unit test for the case where multiple sitemaps are returned. (I'm a bit skeptical that this will work. Won't it start dumping files as you go?)